Romuashi/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require 'colorize'
+
+def data
+  file = File.new('data')
+  file.read
+end
+
+def mean(town, data)
+  return -1 unless data.include?(town)
+
+  search_town(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1 unless data.include?(town)
+
+  search_town(town, data).map { |city| (mean(town, data) - city)**2 }.sum / 12
+end
+
+def search_town(town, data)
+  data.split("\n").each do |line|
+    return line.scan(/\d*\.\d/).map(&:to_f) if line.include?(town)
```

Comment:
This search doesn't work as expected, please adjust it so it'll find only if I provide correct city name

Reply:
![Screenshot from 2021-11-15 21-46-56](https://user-images.githubusercontent.com/10963304/141837553-2bb461e7-99e8-426c-83c1-252d1c6ad6e7.png)